coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,13 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        if ((re.match('[^ ]#', line) or re.match('#[^ ]', line))
+                and len(line) > 2):
+            logging.warning('This comment does not have whitespace' +
+                            ' before or after # in: ' +
+                            repr(line.replace('\n', '')) + '. The line ' +
+                            'should be: ' + repr('# <comment>'))
```

Comment:
maybe add something like "If you didn't mean to make a comment, use a backslash for escaping" as this is the case of `C#` we're talking about right?